compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -1208,6 +1372,32 @@ void mlir::concretelang::python::populateCompilerAPISubmodule(
            [](::concretelang::clientlib::KeySet &keySet) {
              return pybind11::bytes(keySetSerialize(keySet));
            })
+      .def("serialize_lwe_secret_key_as_glwe",
```

Comment:
Should we have a finner API (i.e. get_secret_key, then have a serialize method), I think it would be better.

Reply:
We don't have a need to hold a secret_key python side, so I don't see why it would be useful

Reply:
Yes sure it's ok for this PR, but I think a finner API and user could do more things and have more abilities for manipulating underlying objects.